custom_metrics/css.js

```diff
@@ -0,0 +1,30 @@
+//[css]
+// Uncomment the previous line for testing on webpagetest.org
+
+const CssInJsMap = {
+  'styled-components': document => !!document.querySelector('style[data-styled],style[data-styled-components]'),
+  'radium': document => !!document.querySelector('[data-radium]'),
+  'jss': document => !!document.querySelector('[data-jss]'),
+  'emotion': document => !!document.querySelector('[data-emotion]'),
+  'goober': document => !!document.getElementById('_goober'),
+  'merge-styles': document => !!document.querySelector('[data-merge-styles]'),
+  'jsx': document => !!document.querySelector('style[id*="__jsx-"]'),
+  'aphrodite': document => !!document.querySelector('[data-aphrodite]'),
+  'fela': document => !!document.querySelector('[data-fela-stylesheet]'),
+  'styletron': document => !!document.querySelector('[data-styletron],._styletron_hydrate_'),
+  'react-native-web': document => !!document.querySelector('#react-native-stylesheet'),
+  'glamor': document => !!document.querySelector('[data-glamor]')
+}
+
+return JSON.stringify({
+  'css_in_js': (() => {
+    const usedLibraries = []
+    for (l in CssInJsMap) {
+      if (CssInJsMap[l](document)) {
```

Comment:
Is there a reason why we want these tests to be functions on `document` They seem to be running synchronously, and on the same `document` node as the rest of the JS, so why not just execute inline?

Reply:
Do you think just use ifs inside metric function? The thinking behind a map was easier to extend and less fragile given there is no tests.

Reply:
No, I mean you can still have a map, of ids to booleans, not ids to functions.

Reply:
Gotcha, thanks! I moved map inside the function and inlined all the checks.